docker-compose/versions/camunda-alpha/docker-compose.yaml

```diff
@@ -460,7 +460,7 @@ services:
       RESTAPI_MAIL_FROM_ADDRESS: "noreply@example.com"
       CAMUNDA_MODELER_CLUSTERS_0_ID: "local-zeebe"
       CAMUNDA_MODELER_CLUSTERS_0_NAME: "Local Zeebe instance"
-      CAMUNDA_MODELER_CLUSTERS_0_VERSION: ${CAMUNDA_PLATFORM_VERSION}
+      CAMUNDA_MODELER_CLUSTERS_0_VERSION: ${CAMUNDA_ZEEBE_VERSION}
```

Comment:
For my learning, why is modeler version  = zeebe version?

Reply:
That's the `Local Zeebe instance` version.
Those vars are used to seed the Zeebe config in WM, so the user doesn't need to manually set up the Zeebe cluster info. So this is just a reference to Zeebe deployment.